edge/pkg/edged/edged.go

```diff
@@ -64,13 +65,12 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/util/format"
 	"k8s.io/kubernetes/pkg/kubelet/util/queue"
 	"k8s.io/kubernetes/pkg/kubelet/volumemanager"
-	"k8s.io/kubernetes/pkg/scheduler/schedulercache"
-	kubeio "k8s.io/kubernetes/pkg/util/io"
+	schedulercache "k8s.io/kubernetes/pkg/scheduler/nodeinfo"
 	"k8s.io/kubernetes/pkg/util/mount"
 	"k8s.io/kubernetes/pkg/volume"
 	"k8s.io/kubernetes/pkg/volume/configmap"
-	"k8s.io/kubernetes/pkg/volume/empty_dir"
-	"k8s.io/kubernetes/pkg/volume/host_path"
+	empty_dir "k8s.io/kubernetes/pkg/volume/emptydir"
```

Comment:
its good practice to avoid underscores for package identifiers.

Reply:
Thanks. I will update